packages/language-service/src/completions.ts

```diff
@@ -629,7 +629,7 @@ function angularAttributes(info: AstResult, elementName: string): AngularAttribu
       continue;
     }
     const summary = selectorMap.get(selector) !;
-    const isTemplateRef = hasTemplateReference(summary.type);
+    const isTemplateRef = isStructuralDirective(summary.type);
```

Comment:
nit, for semantics (it [probably] has a template ref, but the directive itself is not one) 

```suggestion
    const hasTemplateRef = isStructuralDirective(summary.type);
```

Reply:
Maybe I can remove the local variable because it's only used in the 'if' condition

Reply:
It runs through a loop though, right?

Reply:
Yes, I will apply your suggestion.